lib/previews/reset_password_preview.rb

```diff
@@ -0,0 +1,5 @@
+class ResetPasswordPreview < ActionMailer::Preview
```

Comment:
Why was this class added?

Reply:
Not completely vital for this feature but I added this class to let us preview the email during development. Happy to get rid of it if you'd like.

Reply:
Isn't that what `letter_opener` is for?

Reply:
Wasn't aware of letter_opener. Thanks for sharing. I've removed this class as well.